manim/mobject/geometry.py

```diff
@@ -287,6 +308,13 @@ def __init__(self, start, end, angle=TAU / 4, **kwargs):
         if angle == 0:
             self.set_points_as_corners([LEFT, RIGHT])
         self.put_start_and_end_on(start, end)
+        
+        if radius==None:
+            center=self.get_arc_center(warning=False)
```

Comment:
Discussion point: I think that warning should be `True` by default. But would like to hear what others think.

Reply:
Warning is True by default. This is a special case because ArcBetweenPoint uses get_arc_center here to insert the correct radius if it was instantiated with an angle.

If this call wouldn't disable the warning then EVERY single ArcBetweenPoint with angle 0 would drop the warning for not being able to find the center. That'd be like... 50 warnings for my recent video? There might be other ways to go around it, but we can't have this get_arc_center trigger the warning.

Reply:
Note: `is None` and not `== None`